Services/DataX.Utilities.JobRunner/StorageUtils.cs

```diff
@@ -0,0 +1,129 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using Microsoft.Azure.Cosmos.Table;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.Storage.File;
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace DataX.Utilities
+{
+    public static class StorageUtils
```

Comment:
Can you add these to existing storage utils instead of creating a new one?

Reply:
Updated. Added these to the existing project DataX.Utility.Blob